manifests/README.md

```diff
@@ -1,52 +1,19 @@
-# Automated Argo workflows
+# OpenShift manifests
 
-If you'd like to automate your Jupyter notebooks using Argo, please use these kustomize manifests. If you follow the steps bellow, your application is fully set and ready to be deployed via Argo CD.
+## Text generation web UI
 
-For a detailed guide on how to adjust your notebooks etc, please consult [documentation](https://github.com/aicoe-aiops/data-science-workflows/blob/master/Automating%20via%20Argo.md)
+The [text-generation-webui.yaml](./text-generation-webui.yaml) manifest builds and deploys [oobabooga's text generation webui](https://github.com/oobabooga/text-generation-webui/), a Gradio app for running Large Language Models.
 
-1. Replace all `<VARIABLE>` mentions with your project name, respective url or any fitting value
-2. Define your automation run structure in the `templates` section of [`cron-workflow.yaml`](./cron-workflow.yml)
-3. Set up `sops`:
+The manifest includes:
 
-   1. Install `go` from your distribution repository
-   2. Setup `GOPATH`
+- a `BuildConfig` to build an image from the repo, using s2i Python
+- a `Deployment` to deploy the built image
+- a `Service` and a `Route** to expose it
 
-      ```bash
-      echo 'export GOPATH="$HOME/.go"' >> ~/.bashrc
-      echo 'export PATH="${GOPATH//://bin:}/bin:$PATH"' >> ~/.bashrc
-      source  ~/.bashrc
-      ```
+### Requirements
 
-   3. Install `sops` from your distribution repository if possible or use [sops GitHub release binaries](https://github.com/mozilla/sops#stable-release)
+**IMPORTANT** about model persistence: the manifest does **NOT** include a `PersistentVolumeClaim` definition. However, the included `Deployment` assumes that a PVC called `llms` exists, and it is expected to be populated with the models to serve, one per directory. The `Deployment` expects to start up with `bloom-1b7`.
 
-   4. Import AICoE-SRE's public key [EFDB9AFBD18936D9AB6B2EECBD2C73FF891FBC7E](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xefdb9afbd18936d9ab6b2eecbd2c73ff891fbc7e):
+This means that in order to use this, a PVC named `llms` **must be created** first with the appropriate content.
```

Comment:
@codificat do you have the definition for the pvc? did you upload the bloom-1b7 model checkpoint into it?

Reply:
@oindrillac I have now added the PVC definition to the YAML manifest and expanded the README with instructions on how to populate it

Reply:
perfect, thanks @codificat!